scio-test/src/it/scala/com/spotify/ScioBenchmark.scala

```diff
@@ -258,18 +259,31 @@ object ScioBenchmark {
 
   private val M = 1000000
   private val K = 1000
-  private val numPartitions = 100
 
-  private def randomUUIDs(sc: ScioContext, n: Long): SCollection[String] =
-    sc.parallelize(Seq.fill(numPartitions)(n / numPartitions))
+  private final case class Elem[T](elem: T)
+
+  private def partitions(
+      n: Long,
+      numPartitions: Int = Runtime.getRuntime
+        .availableProcessors() * numOfWorkers): Iterable[Iterable[Long]] =
```

Comment:
That's the number of processors in `main` not in a worker though.

Reply:
yeah you are right 🤦‍♂️